components/org.wso2.carbon.identity.sso.saml/src/main/java/org/wso2/carbon/identity/sso/saml/exception/IdentitySAML2ECPException.java

```diff
@@ -0,0 +1,9 @@
+package org.wso2.carbon.identity.sso.saml.exception;
```

Comment:
Include license, class comments and public method descriptions.

Reply:
> Include license, class comments and public method descriptions.

license and class comments were added.